### PR TITLE
ipconfig, scrcpy: add Indonesian translation

### PR DESCRIPTION
**Pull request by reinhart1010**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

I accidentally committed my translation of `scrcpy` when preparing to fetch my current working directory from this repo, after my contributions from last Hacktoberfest.

However, I am also working to translate several Windows-specific commands such as `ipconfig`.

### CODE REVIEW

File: pages.id/windows/ipconfig.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Memperbarui alamat IP sebuah adaptor jaringan:
+
+`ipconfig /renew {{adapter}}`
+
+- Mengosongkan alamat-alamat IP yang disetel dalam sebuah adaptor jaringan:
+
+`ipconfig /release {{adapter}}`
+
+- Mengosongkan cache DNS
```
Comment (bl-ue) [inline pages.id/windows/ipconfig.md:22]:
```suggestion
- Mengosongkan cache DNS:
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Nice, thank you for the translations.  Don't worry about the old commits ;)

Maybe it would be good to translate the variables like `path/to/file` 🤔

File: pages.id/windows/ipconfig.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# ipconfig
+
+> Menampilkan dan mengatur konfiigurasi jaringan dalam sistem operasi Windows.
```
[the remark this answers is not in the document]
Comment (reinhart1010) [reply pages.id/windows/ipconfig.md]: Resolved in https://github.com/tldr-pages/tldr/commit/e3b08f1d13862e7dc290fde4b0c704f97cf7a6f7

File: pages.id/windows/ipconfig.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Menunjukkan daftar adaptor jaringan secara lengkap:
+
+`ipconfig /all`
+
+- Memperbarui alamat IP sebuah adaptor jaringan:
+
+`ipconfig /renew {{adapter}}`
+
+- Mengosongkan alamat-alamat IP yang disetel dalam sebuah adaptor jaringan:
```
[the remark this answers is not in the document]
Comment (reinhart1010) [reply pages.id/windows/ipconfig.md:18]: The original description refers to "IP Address**es**", so yes it's the plural form of "alamat".

File: pages.id/common/scrcpy.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`scrcpy --rotation {{0|1|2|3}}`
+
+- Tunjukkan indikator sentuhan pada perangkat fisik:
+
+`scrcpy --show-touches`
+
+- Rekam tampilan layar perangkat ke dalam file video tertentu:
+
+`scrcpy --record {{path/to/file.mp4}}`
```
Comment (navarroaxel) [inline pages.id/common/scrcpy.md:28]: could you translate `path/to/file` to Indonesian? the same for `path/to/directory`.

File: pages.id/windows/ipconfig.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`ipconfig /release {{adapter}}`
```
Comment (navarroaxel) [inline pages.id/windows/ipconfig.md:20]: Please translate the `adapter` work to Indonesian to help non-English speaker.

File: pages.id/windows/ipconfig.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (bl-ue) [reply pages.id/windows/ipconfig.md]: So my guess was correct? 😁

File: pages.id/windows/ipconfig.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (bl-ue) [reply pages.id/windows/ipconfig.md:18]: Oh, I see! Interesting 🤔

File: pages.id/windows/ipconfig.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (reinhart1010) [reply pages.id/windows/ipconfig.md]: Yes

File: pages.id/common/scrcpy.md (hunk 4 above)
Comment (reinhart1010) [reply pages.id/common/scrcpy.md]: See https://github.com/tldr-pages/tldr/commit/012ad23564d66e4d780288d0fe388ea059fdf425

File: pages.id/windows/ipconfig.md (hunk 5 above)
Comment (reinhart1010) [reply pages.id/windows/ipconfig.md]: See https://github.com/tldr-pages/tldr/pull/6045/commits/46a932dc57c0f544c126cc1c8cc6b52f60539494

`adaptor` is considered [a standard word](https://kbbi.kemdikbud.go.id/entri/adaptor) of the Indonesian language, so I don't translate `adaptor` further from `adapter`.

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @reinhart1010

**Review by sbrl — APPROVED** [review]
Thanks, @reinhart1010!

Merging, since @bl-ue's comments appear to be addressed.

**Comment by sbrl** [comment]
Oops, forgot to clear the extended commit description there

**Pull request merged**